ui/web/index.html

```diff
@@ -3,10 +3,323 @@
+                </td>
+                <td>max_estimate
+                    <select name="max_estimate" id="maxEst">
+                        <option value=""></option>
+                        <option value="A">A</option>
+                        <option value="B">B</option>
+                        <option value="C">C</option>
+                        <option value="D">D</option>
+                        <option value="E">E</option>
+                        <option value="F">F</option>
+                        <option value="G">G</option>
+                        <option value="H">H</option>
+                        <option value="I">I</option>
+                        <option value="J">J</option>
+                        <option value="K">K</option>
+                        <option value="L">L</option>            
+                    </select>
+                </td>
+                </td>
+            </tr>
+        </table>
+    </div> 
+
+    </fieldset>
+
+    <fieldset>
+
+        <legend>Load</legend>
+
+        <div id="place">
+
+            <input type="submit" name="but" id="loadTasksButton" value="Load">
+
+        </div>
+
+    </fieldset>
+    <fieldset>
+
+        <legend>Print</legend>
+
+        <div id="schedule">
+
+            <table border="0" widtd="100%">
+                <tr>
+                    <td>Tasks
+                        <select name="tasks" id="tasks">
+                            <option value=""></option>
+                            <option value="0">Task 1</option>
+                            <option value="1">Task 2</option>
+                            <option value="2">Task 3</option>
+                            <option value="3">Task 4</option>
+                            <option value="4">Task 5</option>
+                            <option value="5">Task 6</option>
+                            <option value="6">Task 7</option>
+                            <option value="7">Task 8</option>
+                            <option value="8">Task 9</option>
+                            <option value="9">Task 10</option>
+                            <option value="10">Task 11</option>
+                            <option value="11">Task 12</option>            
+                        </select>                       
+                    </td>
+                    <td>
+                        <input type="button" value="Print" id="loadPrint">
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <div id="chartdiv" style="height:500px;width:300px; "></div>
+                    </td>
+                </tr>
+                </table>
+    </fieldset>
+
+    <script>  
+
+        var columns_mapping = {};
+        var requestBody = {
+            columns_mapping
+        };
+        var applicationContext = {};
+        var itemNumber;
+        var tasks = [];
+        var inquiryPoints = {"numberOfPoints": 50, 
+        tasks
+        };
```

Comment:
This code:
```
        var itemNumber;
        var tasks = [];
        var inquiryPoints = {"numberOfPoints": 50, 
        tasks
        };
```
can be moved more close to its usage.

Reply:
Recommended changes made